### ISSUE TITLE
Using Add-Meta-Tags with Visual Composer

### ISSUE THREAD

**Issue by gnotaras** [gnotaras/wordpress-add-meta-tags#24]
I'm opening this ticket for open discussion and feedback regarding interoperability between Add-Meta-Tags and Visual Composer. Feel free to post your experience.

**Comment by gnotaras** [comment]
I have received a report about descriptions not being automatically generated from the post's content. However, adding a custom description works as expected.

As a workaround, I'm posting the following snippet that could be added in the theme's `functions.php` file. Please try this on a staging web site on which you have shell or ftp access so as to be able to revert any changes outside of WordPress' web interface.

```php
function amt_force_post_content_filtering($post_content, $post) {
    return do_shortcode( $post_content );
}
add_filter('amt_get_the_excerpt_initial_content', 'amt_force_post_content_filtering', 10, 2);
```

**Comment by gnotaras** [comment]
An option to enforce the expansion of shortcodes has been added in 2.10.8. Enabling this option should fix any issues.

**Comment by gnotaras** [comment]
Same fix for #30

**Comment by gnotaras** [comment]
Note: no more research will be done for this issue.
